azure-devops/azext_devops/dev/common/services.py

```diff
@@ -301,18 +302,33 @@ def resolve_instance_project_and_repo(
         project_required=True,
         repo=None,
         repo_required=False):
+    init_telemetry()
+    vsts_tracking_data.properties['OrgPresentInCommand'] = organization is not None
```

Comment:
Should we define all properties constants at one place. till now we had only one function where everything was getting logged.
This will get scattered in the code and will be hard to find later.

Reply:
sounds good..
pls take a look at the names (key) also  ..

Reply:
They look good to me.